### PR TITLE
Add Cohere RAG tutorial link

### PR DESCRIPTION
**Pull request by tianjing-li**
### Description
_Describe what this change achieves._

In the RAG pipeline section, adds a link to the Cohere tutorial setup for RAG

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _search-plugins/conversational-search.md [hunk 1]
```diff
@@ -226,9 +226,11 @@ The Memory API responds with the following:
 
 RAG is a technique that retrieves documents from an index, passes them through a seq2seq model, such as an LLM, and then supplements the static LLM information with the dynamically retrieved data in context.
 
-As of OpenSearch 2.11, the RAG technique has only been tested with OpenAI models and the Anthropic Claude model on Amazon Bedrock.
+As of OpenSearch 2.11, the RAG technique has only been tested with OpenAI models, the Anthropic Claude model on Amazon Bedrock, and Cohere command models. 
 {: .warning}
 
+Setting up Cohere's command model to enable RAG with chat requires modified request bodies than shown below, please refer to the [Cohere RAG Tutorial](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/conversational_search/conversational_search_with_Cohere_Command.md) for more details.
```
Comment (kolchfa-aws) [inline _search-plugins/conversational-search.md:232]: I'm planning to migrate the tutorials that live in ML Commons to the documentation. For now, we can link to GitHub, but this link will change once the migration is complete.
```suggestion
Configuring the Cohere Command model to enable RAG requires using a post-processing function to transform the model output. For more information, see [Cohere RAG Tutorial](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/conversational_search/conversational_search_with_Cohere_Command.md).
```

**Review by kolchfa-aws — APPROVED** [review]
LGTM with comments.

File: _search-plugins/conversational-search.md [hunk 2]
```diff
@@ -226,9 +226,11 @@ The Memory API responds with the following:
 
 RAG is a technique that retrieves documents from an index, passes them through a seq2seq model, such as an LLM, and then supplements the static LLM information with the dynamically retrieved data in context.
 
-As of OpenSearch 2.11, the RAG technique has only been tested with OpenAI models and the Anthropic Claude model on Amazon Bedrock.
+As of OpenSearch 2.11, the RAG technique has only been tested with OpenAI models, the Anthropic Claude model on Amazon Bedrock, and Cohere command models.
```
[the remark this answers is not in the document]
Comment (tianjing-li) [reply _search-plugins/conversational-search.md]: Updated this

File: _search-plugins/conversational-search.md (hunk 1 above)
Comment (tianjing-li) [reply _search-plugins/conversational-search.md]: SGTM - Updated the sentence

File: _search-plugins/conversational-search.md [hunk 3]
```diff
@@ -226,6 +226,17 @@ POST _bulk
...
 {% include copy-curl.html %}
 
+## RAG pipeline
+
+RAG is a technique that retrieves documents from an index, passes them through a seq2seq model, such as an LLM, and then supplements the static LLM information with the dynamically retrieved data in context.
+
+As of OpenSearch 2.12, the RAG technique has only been tested with OpenAI models, the Anthropic Claude model on Amazon Bedrock, and Cohere Command models. 
+{: .warning}
+
+Configuring the Cohere Command model to enable RAG requires using a post-processing function to transform the model output. For more details, you can refer to the [Cohere RAG Tutorial](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/conversational_search/conversational_search_with_Cohere_Command.md).
```
Comment (kolchfa-aws) [inline _search-plugins/conversational-search.md:236]:
```suggestion
Configuring the Cohere Command model to enable RAG requires using a post-processing function to transform the model output. For more details, see the [Cohere RAG Tutorial](https://github.com/opensearch-project/ml-commons/blob/2.x/docs/tutorials/conversational_search/conversational_search_with_Cohere_Command.md).
```

**Review by natebower — APPROVED** [review]
@kolchfa-aws @tianjing-li Just one word swap. Thanks!

**Pull request merged**